packages/app-bridge-app/src/registries/api/GetAccountId.spec.ts

```diff
@@ -0,0 +1,12 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { describe, expect, it } from 'vitest';
+
+import { getAccountId } from './GetAccountId.ts';
+
+describe('GetAccountId', () => {
+    it('should return correct method name', () => {
+        const currentUser = getAccountId();
+        expect(currentUser.name).toBe('getAccountId');
```

Comment:
I guess instead of `currentUser` it should be `getAccountIdMethod`?

Same goes for 
`packages/app-bridge/src/registries/api/CreateAsset.spec.ts`
and
`packages/app-bridge-app/src/registries/api/CreateAsset.spec.ts`